code/src/index.css

```diff
@@ -11,3 +11,99 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.header {
+  display: grid;
+  grid-template-columns: 1fr 300px 1fr;
```

Comment:
Nice that you are working with grids. I have never used it so impressed you have learned that! 🥇